src/service/resources-impl.js

```diff
@@ -758,20 +768,19 @@ export class Resources {
           if (isScrollingStopped) {
             // These requests will be executed in the next animation cycle and
             // adjust the scroll position.
-            resize = false;
             scrollAdjSet.push(request);
           } else {
             // Defer till next cycle.
             this.requestsChangeSize_.push(request);
           }
+          continue;
         } else if (iniBox.bottom >= docBottomOffset ||
                       box.bottom >= docBottomOffset) {
           // 5. Elements close to the bottom of the document (not viewport)
           // are resized immediately.
           resize = true;
         } else if (diff < 0) {
           // 6. The new height is smaller than the current one.
```

Comment:
Do nothing in this `else if` condition. Do we want to delete it? or just leave the comment here?